MSAL/test/automation/tests/interactive/MSALB2CInteractiveTests.m

```diff
@@ -118,7 +118,7 @@ - (void)testInteractiveB2CLogin_withEmbeddedWebView_withoutLoginHint_withSigninP
     MSIDAutomationTestRequest *request = [self.class.confProvider defaultAppRequest:self.testEnvironment targetTenantId:self.primaryAccount.targetTenantId];
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->